ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Selection-Tool/Lasso-Selection-Tool/lasso-tool.spec.ts

```diff
@@ -0,0 +1,258 @@
+/* eslint-disable no-magic-numbers */
+import { Page, test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  openFileAndAddToCanvas,
+  clickOnAtom,
+  dragMouseTo,
+  selectTopPanelButton,
+  TopPanelButton,
+  drawBenzeneRing,
+  getCoordinatesTopAtomOfBenzeneRing,
+  BondType,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  getControlModifier,
+  BondTool,
+  selectNestedTool,
+  delay,
+  DELAY_IN_SECONDS,
+  waitForIndigoToLoad,
+} from '@utils';
+import { getAtomByIndex } from '@utils/canvas/atoms';
+import { getBondByIndex } from '@utils/canvas/bonds';
+
+test.describe('Lasso Selection tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+    await waitForIndigoToLoad(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  const xDelta = 30;
+  const yDelta = 60;
+  const modifier = getControlModifier();
+
+  const selectLasso = async (page: Page) => {
+    await page.getByTestId('select-rectangle').click();
+    await page.getByTestId('select-lasso').click();
+  };
+
+  async function selectObjects(page: Page, xAxis: number, yAxis: number) {
+    const point = await getCoordinatesOfTheMiddleOfTheScreen(page);
+    await page.mouse.move(point.x - xAxis, point.y - yAxis);
+    await page.mouse.down();
+    await page.mouse.move(point.x + xAxis, point.y - yAxis);
+    await page.mouse.move(point.x + xAxis, point.y + yAxis);
+    await page.mouse.move(point.x - xAxis, point.y + yAxis);
+    await page.mouse.up();
+    return point;
+  }
+
+  async function clickCanvas(page: Page) {
+    await page.mouse.click(300, 200);
+  }
+
+  test('Selection of atom/bond/molecule', async ({ page }) => {
+    /*
+     * Test case: EPMLSOPKET-1338
+     * Description: Hover and selection of atom/bond/molecule
+     */
+    await openFileAndAddToCanvas('KET/two-benzene-with-atoms.ket', page);
+    const atomPoint = await getAtomByIndex(page, { label: 'C' }, 0);
+    await page.mouse.move(atomPoint.x, atomPoint.y);
+    await page.mouse.click(atomPoint.x, atomPoint.y);
+    await clickCanvas(page);
```

Comment:
I propose to split this test into multiple – click for atom, click for bond, click for multiple. In addition, I want to mention, that you have no assertions after clicking on atom/bond. So we need to check somehow, whether atom is selected after click

Reply:
will add the screenshot for each click for atom, click for bond, click for multiple